### PR TITLE
+ Integrated Travis CI with Github repo

### PR DESCRIPTION
**Pull request by Oleksandr-Adamchuk**
Configured Travis CI to run RuboCop tests ans rspecs tests.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Metrics/MethodLength:
+  Exclude:
+    - 'app/helpers/*'
+
+Metrics/ClassLength:
+  Exclude:
+    - 'app/models/*'
+
+Metrics/LineLength:
+  Max: 120
```
Comment (romaslmd) [inline .rubocop.yml]: isn't that too many?
 90 fair enough

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - bundle exec rake db:setup
+  - bundle exec rake spec
+
+services:
+  - postgresql
+
+before_script:
+    - cp config/database.yml.example config/database.yml
+    - bundle exec rubocop
+    - psql -c 'create database heros_team_app_test;' -U postgres
```
Comment (romaslmd) [inline .travis.yml]: it's probably better to created DB using rails command, as manual creation may bring inconsistency in data encoding

or in this case just ensure that DB encodings match for rails app and default for `CREATE DATABASE` command

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -36,11 +36,10 @@
 end
 
 RSpec.configure do |config|
-
   config.before(:all) do
-  FactoryBot.reload
-  
-# add `FactoryBot` methods
+    FactoryBot.reload
+  end
```
Comment (romaslmd) [inline spec/rails_helper.rb]: could you clean this up a bit
```
# includes first
config.include ...

# hooks later
config.before() {}
```

or move to different files

File: .rubocop.yml (hunk 1 above)
Comment (Oleksandr-Adamchuk) [reply .rubocop.yml]: There are a lot of comment strings in rails files, that have length between 90 and 100 characters. Can we set Max to 100?

File: .travis.yml (hunk 2 above)
Comment (Oleksandr-Adamchuk) [reply .travis.yml]: Setup DB creating via rails command.

File: spec/rails_helper.rb (hunk 3 above)
Comment (Oleksandr-Adamchuk) [reply spec/rails_helper.rb]: Cleaned

**Pull request merged**